saltedge-compliance-connector-ob-java/compliance-connector-ob-sdk/src/main/java/com/saltedge/connector/ob/sdk/api/models/request/ConsentsRevokeRequest.java

```diff
@@ -31,10 +31,19 @@
 @JsonIgnoreProperties
 public class ConsentsRevokeRequest extends PrioraBaseRequest {
 
+    /**
+     * The consent identifier assigned to the created resource.
+     */
+    @NotBlank
+    @JsonProperty(SDKConstants.KEY_CONSENT_ID)
+    public String consentId;
```

Comment:
consentId is not a part of request payload. it is a part of query path. please read docs carefully